enhancements/windows-containers/health-management.md

```diff
@@ -66,18 +66,26 @@ configuration of a Windows instance, to ensure that it can become a worker Node.
 
 To accomplish the goals listed in this enhancement, I am proposing that WMCB be converted into a daemon,
 Windows Instance Config Daemon (WICD). WICD will have the following responsibilities:
-* WICD maintains current responsibilities of bootstrapping kubelet through the `bootstrap` command.
+* As part of the `bootstrap` phase, WICD will configure the [containerd](container-runtime-containerd.md) runtime and 
+  start it as a Windows service, if its configuration is detailed in the services ConfigMap.
+* WICD will also maintain current responsibilities of starting kubelet as a service through the `bootstrap` command. 
+  WICD will fully configure kubelet in one shot, as everything required will be available in the services ConfigMap.
```

Comment:
How is this possible when using Docker? Don't you need to configure hybrid-overlay and the reconfigure kubelet for CNI?

Reply:
Added `As the shift to containerd is imminent, this enhancement is written considering containerd as the only supported container runtime.` to the summary, as discussed by the team.

Reply:
You also need to mention somewhere that you will be using the Docker runtime flag introduced by the containerd enhancement to develop during the 5.y.z cycle and this will be dropped in the 6.0.0 timeframe.